apps/user/components/tips/index.coffee

```diff
@@ -7,6 +7,7 @@ module.exports = ->
   tips = new Backbone.Collection TIPS
 
   tips.each (tip) ->
+    return if Cookies.get(tip.id) 
     new TipView
       el: $(".js-tip[data-id='#{tip.id}']")
       model: tip
```

Comment:
Ah, yes you're right. Was probably running into a different issue that ended up showing a tip when it shouldn't. Closing.